_includes/configuration/rtc_client_connected.md

```diff
@@ -12,7 +12,7 @@ Required plugin
 {% endif %}
 
 Type
-: Function (Returns a [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise))
```

Comment:
There's quite a number that have been missed here:

Reply:
So looking at that list, only 4 out of 8 have been changed.

Reply:
those are promises. these ones are callbacks

Reply:
Yes the problem is it was added to all 8 functions, but only 4 of them needed it. I didn't catch this in the pre-release review.